packages/command-palette/src/commands/use-single-site-commands.tsx

```diff
@@ -528,15 +530,25 @@ const useSingleSiteCommands = ( { navigate, currentRoute }: useCommandsParams ):
 				),
 				_x( 'audit log', 'Keyword for the Open activity log command', __i18n_text_domain__ ),
 			].join( ' ' ),
-			callback: commandNavigation( '/activity-log/:site' ),
+			callback: commandNavigation(
+				`${
+					shouldUseWpAdmin
+						? 'https://jetpack.com/redirect/?source=calypso-activity-log&site='
```

Comment:
This redirects me to Calypso (`https://wordpress.com/activity-log/:site`). I was expecting a redirection tp WP Admin.

I tested with an Atomic site and the "classic interface" setting enabled.